src/jdk.crypto.cryptoki/share/native/libj2pkcs11/pkcs11wrapper.h

```diff
@@ -76,6 +82,8 @@
 #define CKA_NETSCAPE_TRUST_EMAIL_PROTECTION     (CKA_NETSCAPE_TRUST_BASE + 11)
 #define CKA_NETSCAPE_DB                         0xD5A0DB00
 #define CKM_NSS_TLS_PRF_GENERAL                 0x80000373
+#define CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE      (CKA_NETSCAPE_BASE + 25)
+#define CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_DH   (CKA_NETSCAPE_BASE + 26)
```

Comment:
Please line up the values where possible. Either just one space after `CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_DH` or fold the value onto a new line.